schema.cc

```diff
@@ -102,21 +102,30 @@ std::ostream& operator<<(std::ostream& os, ordinal_column_id id)
     return os << static_cast<column_count_type>(id);
 }
 
-std::unique_ptr<dht::i_partitioner> default_partitioner;
+std::map<std::tuple<sstring, unsigned, unsigned>, std::unique_ptr<dht::i_partitioner>> cached_partitioners;
+sstring default_partitioner_name = "org.apache.cassandra.dht.Murmur3Partitioner";
```

Comment:
The *previous* patch claimed it was bout moving the default partitioner to schema.cc - but now I see it in this patch :-)

Reply:
This is the next step. Previous patch moved default partitioner from i_partitioner to schema and this patch changes the way it's stored. It used to be stored as i_partitioner object but now it will be just a name and ignore_msb value. The i_partition object itself will be cached in cached_partitioners.